### PR TITLE
Emit correct commit hash

### PR DESCRIPTION
**Pull request by TheRealPiotrP**

### CODE REVIEW

File: .github/workflows/regenerate.yaml [hunk 1]
```diff
@@ -55,10 +55,7 @@ jobs:
       - run: yarn install --frozen-lockfile
 
       - name: Generate commit_hash.sh
-        run: echo $'COMMIT_HASH=${COMMIT_HASH}\n' >> downloaded/commit_hash.sh
-        shell: bash
-        env:
-          COMMIT_HASH: ${{ github.event.inputs.openapi_grpc_sha }}
+        run: echo $'COMMIT_HASH=${{ github.event.inputs.openapi_grpc_sha }}\n' > ./downloaded/commit_hash.sh
```
Comment (authereal) [inline .github/workflows/regenerate.yaml:58]: just for my education, what does the leading $ do in `echo $'`?

File: downloaded/commit_hash.sh [hunk 2]
```diff
@@ -1 +1,3 @@
 COMMIT_HASH=7252b1aeb598c51308d44a0e3ca48aa84b036b59
+COMMIT_HASH=${COMMIT_HASH}
```
Comment (authereal) [inline downloaded/commit_hash.sh:2]: I'm afraid I don't really understand this change. is it intentional?

**Review by authereal — CHANGES_REQUESTED** [review]

File: .github/workflows/regenerate.yaml (hunk 1 above)
Comment (TheRealPiotrP) [reply .github/workflows/regenerate.yaml:58]: handles `\n` as a newline. Without it, the character `n` was being emitted after the sha value

File: downloaded/commit_hash.sh (hunk 2 above)
Comment (TheRealPiotrP) [reply downloaded/commit_hash.sh:2]: asking aserto-bot...

**Pull request merged**